detection_rules/etc/version.lock.json

```diff
@@ -1,178 +1,410 @@
 {
   "000047bb-b27a-47ec-8b62-ef1a5d2c9e19": {
+    "previous": {
+      "8.2": {
+        "rule_name": "Attempt to Modify an Okta Policy Rule",
+        "sha256": "dedf2a77f86a3ecebeba40e8a1f54e713510e09384f2ca228c8adb9cc6322490",
+        "type": "query",
+        "version": 8
+      }
+    },
     "rule_name": "Attempt to Modify an Okta Policy Rule",
     "sha256": "dedf2a77f86a3ecebeba40e8a1f54e713510e09384f2ca228c8adb9cc6322490",
     "type": "query",
-    "version": 8
+    "version": 9
   },
   "00140285-b827-4aee-aa09-8113f58a08f3": {
     "min_stack_version": "7.16",
+    "previous": {
+      "8.2": {
+        "rule_name": "Potential Credential Access via Windows Utilities",
+        "sha256": "3c01d95323f7c5eb6e183e2123acd3cdd13019c938cea3a4331856c7fc486ca7",
+        "type": "eql",
+        "version": 8
+      }
+    },
```

Comment:
Did this overwrite the old previous? Where is the 7.16 previous?

Reply:
These are min_stacked but have not been forked yet.

```
  "00140285-b827-4aee-aa09-8113f58a08f3": {
    "min_stack_version": "7.16",
    "rule_name": "Potential Credential Access via Windows Utilities",
    "sha256": "3c01d95323f7c5eb6e183e2123acd3cdd13019c938cea3a4331856c7fc486ca7",
    "type": "eql",
    "version": 8
```